app/notifications/receive_notifications.py

```diff
@@ -86,7 +87,35 @@ def create_inbound_mmg_sms_object(service, json):
 @receive_notifications_blueprint.route('/notifications/sms/receive/firetext', methods=['POST'])
 def receive_firetext_sms():
     post_data = request.form
-    current_app.logger.info("Received Firetext notification form data: {}".format(post_data))
+
+    potential_services = dao_fetch_services_by_sms_sender(post_data['destination'])
```

Comment:
The two endpoints to receive_inbound are almost identical, we could refactor them a bit to reuse the code.